### DISCUSSION TITLE
code block fold fail

### DISCUSSION THREAD

**Discussion by github-lym** [chipzoller/hugo-clarity#274] [General]
codeMaxLines can work.
But it always show full code block.
Any  idea please?

Thx...
![sn](https://user-images.githubusercontent.com/69377506/159661546-2a0c4ee8-4c43-4bd7-9a47-50862c8c3946.PNG)
.

**Comment by chipzoller** [comment]
Not sure I understand. Please explain more.

**Reply by rootwork** [reply]
This is being tracked in #280 now.

**Comment by chipzoller** [comment]
This issue is now resolved.
